CHANGELOG.md

```diff
@@ -0,0 +1,140 @@
+
+## Release Notes - Identity Service
+
+### Version: <em>identity-service-1.0</em>
+
+<h2>Epic
+</h2>
+<ul>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-134'>AUTH-134</a>] -         Identity Service 1.0
+</li>
+</ul>
+
+<h2>Story
+</h2>
+<ul>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-65'>AUTH-65</a>] -         Integrate code into APS to implement the use of the internal auth token
```

Comment:
This has been delivered to APS, hence probably better mentioned in its Release Notes rather than Identity Service ones.

Reply:
I removed AUTH-65